code/datums/greyscale/json_configs/messyworn_shirt.json

```diff
@@ -0,0 +1,10 @@
+{
+    "messyworn_shirt": [
+        {
+            "type": "icon_state",
+            "icon_state": "worn_messy",
+            "blend_mode": "overlay",
+            "color_ids": [ 1 ]
+        }
+    ]
+}
```

Comment:
This could be a state in the same .json as wellworn_shirt and wornout_shirt, as they have the same number of layers and color_ids... it would conflict with your change to select icon based on the preview, but I have comments on that anyways